### PR TITLE
fix(cxl-lumo-styles): add CSS `top` property to `headroom`

### PR DESCRIPTION
**Pull request by saas786**
Issue: #196 
Inspired by:
- [WordPress/wporg-news-2021@`20f1bde` (#178)](https://github.com/WordPress/wporg-news-2021/pull/178/commits/20f1bde806e27cace085d8a53709e6d7e203eb6f)
- [WordPress/wporg-news-2021@`3106b25` (#178)](https://github.com/WordPress/wporg-news-2021/pull/178/commits/3106b259751b48321145f9bbf82e342b8ea2d7a8)
- https://github.com/WordPress/WordPress/commit/e32d9ec5226ec01e4a2de3cec4a4da0582d1d52c

### CODE REVIEW

**Comment by kylebrodeur** [comment]
Task linked: [CU-344dzjz Sticky header (headroom) partially hidden behind wp-admin-bar #196](https://app.clickup.com/t/344dzjz)

**Review by anoblet — APPROVED** [review]

File: packages/cxl-lumo-styles/scss/global.scss [hunk 1]
```diff
@@ -79,6 +79,7 @@ ul {
  * headroom.js
  */
 .headroom {
+  top: var(--wp-admin--admin-bar--height, 0);
```
Comment (lkraav) [inline packages/cxl-lumo-styles/scss/global.scss:82]: Was this tested in logged out mode?

I don't see it being scoped in any way to admin-bar-only?

**Review by lkraav — CHANGES_REQUESTED** [review]

File: packages/cxl-lumo-styles/scss/global.scss (hunk 1 above)
Comment (saas786) [reply packages/cxl-lumo-styles/scss/global.scss:82]: > Was this tested in logged out mode?
> 
> I don't see it being scoped in any way to admin-bar-only?

After: https://github.com/WordPress/WordPress/commit/e32d9ec5226ec01e4a2de3cec4a4da0582d1d52c
Its automatically handled.

Logged out:
logged-out-firefox_NTCdYkocxe

Logged in:
logged-in-firefox_nuz0HStY54

Comment (lkraav) [reply packages/cxl-lumo-styles/scss/global.scss:82]: Ah yeah, CSS var fallback should work.

**Review by lkraav — APPROVED** [review]

**Pull request merged**